lib/Pipedrive.js

```diff
@@ -35,15 +34,7 @@
 		options = _.extend({}, defaults, options);
 		options.strictMode = !!options.strictMode;
```

Comment:
Is strictMode still needed?

Reply:
it's not related to this PR, but yes, strict_mode param should collapse `_id` fields to int values in some endpoints (like deals), instead of using full objects without strict_mode